hub/saml-engine/src/main/java/uk/gov/ida/saml/core/transformers/outbound/decorators/NullResponseAssertionSigner.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.ida.saml.core.transformers.outbound.decorators;
+
+import org.opensaml.saml.saml2.core.Response;
+
+public class NullResponseAssertionSigner extends ResponseAssertionSigner {
```

Comment:
Can we make this a `ThrowingResponseAssertionSigner` to make sure we don't end up using it accidentally?